Cargo.toml

```diff
@@ -8,6 +8,7 @@ authors = ["Theia Vogel <theia@vgel.me>"]
 lazy_static = "1.4.0"
 thiserror = "1.0.30"
 smallvec = "1.8.0"
+anyhow = "1.0.58"
```

Comment:
Because this is a library I'd prefer we expose real error enums using `thiserror` instead of using `anyhow`.